### PR TITLE
CI: Add TravisCI config file and .ci/setup.sh

### PR DESCRIPTION
**Pull request by gorozco1**
Add TravisCI support by creating the required ".travis.yml"
configuration file.

This initial commit just run checkcommits.

Fixes: #15

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
@gorozco1 You have a Travis failure, which looks like it might be because Travis is trying to sync up the go sources for this repo, that has no go source in it. I'm guessing you need go enabled to have the scripts build all the sub-packages, but Travis thus thinks this must be a Go based repo. Just a guess. Could be awkward :-(

**Comment by gorozco1** [comment]
@grahamwhaley yes I noticed that :-/ this is odd, probably because in a previous commit it has setup the language to `go`, however in this new commit that is not the case... @jodh-intel do you have any clue?

File: .ci/setup.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+set -e
+
+source $(dirname "$0")/go-env.sh
+
+go_version=1.7
+
+go_tarball="go${go_version}.linux-amd64.tar.gz"
+curl -L -O "https://storage.googleapis.com/golang/$go_tarball"
+tar xvf $go_tarball 1>/dev/null
```
Comment (jcvenegas) [inline .ci/setup.sh]: `"$go_tarball"`

**Comment by jcvenegas** [comment]
lgtm , just a little comment , and travis is working  :D

**Review by jodh-intel — APPROVED** [review]

**Pull request merged**